pandas/core/generic.py

```diff
@@ -11191,7 +11192,26 @@ def block_accum_func(blk_values):
             result = result.T if hasattr(result, "T") else result
             return result
 
-        result = self._mgr.apply(block_accum_func)
+        if isinstance(self.values, ExtensionArray):
+            result = self.values._accumulate(name, skipna, **kwargs)
+        else:
+            result = self._mgr.apply(block_accum_func)
```

Comment:
I think we always want `self._mgr.apply(block_accum_func)`.

We'll just update `block_accum_func` to handle EAs.  something like

```
if is_extension_array_dtype(self.dtype):
    result = self.array._accumulate(name, skipna, **kwargs)
else:
    ...
```

Reply:
Thanks, I was sensing something like this, but I somehow was not finding the accessor of the array